app/robots/iCubGenova09/openXRHeadsetParameters.ini

```diff
@@ -51,18 +52,22 @@ relative_position  (0.0, 0.0, 0.0)
 relative_rotation  (0.0, -0.2, 0.0)
 
 [GUI_0]
-width              0.15
-height             0.15
-x                  0.16
-y                 -0.2
-z                 -0.5
+width              1.50
+height             1.50
+x                 -0.034
+y                  0.0
+z                 -0.99
+visibility         left
+port_id            leftEyeHand
 
 [GUI_1]
-width              0.15
-height             0.15
-x                 -0.16
-y                 -0.2
-z                 -0.5
+width              1.50
+height             1.50
+x                  0.034
+y                  0.0
+z                 -0.99
+visibility         right
+port_id            rightEyeHand
```

Comment:
Since now we have custom port names, we need to update also the connections in one of the sub-applications